src/DonationForms/resources/app/form/MultiStepForm/components/SubmitButton.tsx

```diff
@@ -3,14 +3,12 @@ import {__} from '@wordpress/i18n';
 /**
  * @since 3.0.0
  */
-export default function SubmitButton({
-    isSubmitting,
-    submittingText = __('Submitting…', 'give'),
-    buttonText = __('Donate Now', 'give'),
-}) {
+export default function SubmitButton({isSubmitting, submittingText = __('Submitting…', 'give')}) {
+    const {donateButtonCaption} = window.givewp.form.hooks.useDonationFormSettings();
```

Comment:
@JoshuaHungDinh i'm getting a type error on this, please add to `PropTypes` in `src/DonationForms/resources/app/store/form-settings/index.tsx`

```
donateButtonCaption: string;
```

Reply:
Resolved: 0eb16bef6c58361a0ddd1b855bd8398c8b0499eb